src/sql/workbench/contrib/dashboard/browser/pages/databaseDashboardPage.component.ts

```diff
@@ -57,4 +65,17 @@ export class DatabaseDashboardPage extends DashboardPage implements OnInit {
 		this.init();
 		this._breadcrumbService.setBreadcrumbs(BreadcrumbClass.DatabasePage);
 	}
+
+	protected createTaskbar(parentElement: HTMLElement): void {
```

Comment:
This seems like a better place to add the actions in - not in the base class. 

Also have you thought about how extensions would contribute actions to the taskbar? Ideally the JSON contribution for the tasks-widget would stay basically the same.